bus_tool_index_file_creation.sh

```diff
@@ -0,0 +1,7 @@
+#!/bin/bash
+
+kb ref -i index_file_bustool/transcriptome.idx \
```

Comment:
The index should be named using the reference naming convention. In this case GRCh38_idx